### ISSUE TITLE
mautrix-telegram cannot display some messages, but displays "This message is not supported on your version of Mautrix-Telegram. Please check https://github.com/mautrix/telegram or ask your bridge administrator about possible updates.".

### ISSUE THREAD

**Issue by dreirund** [mautrix/telegram#942]
I recently had that when some people wrote a message in a Telegram group that I participate in with mautrix-telegram, instead of seeing the message I just see

> *his message is not supported on your version of Mautrix-Telegram. Please check https://github.com/mautrix/telegram or ask your bridge administrator about possible updates.*

The message source of one of those messages as seen by my Matrix client is (I have censored some potential sensitive information):  
```
{
  "content": {
    "body": "> <@[censored]:[censored]> > *Insofern wäre ich besonders an Erfahrungen interessiert, die weitere Wege beschreiben, mit Dingen umzugehen, die ihm\\* nicht passen. Bitte teilt sie.*\n> \n> Einer Gruppe ist bewusst, dass es auch Menschen gibt die gut Missstände wahrnehmen können aber schwer eine neue Initiative starten können, und hat sich auf Strukturen geeinigt die auch einfach Wahrnehmungen und Feedback entgegennimmt und das dann an anderer Stelle diskutiert wird und ggf. von dort Anregungen zu Änderungen kommen.\n\nThis message is not supported on your version of Mautrix-Telegram. Please check https://github.com/mautrix/telegram or ask your bridge administrator about possible updates.",
    "external_url": "https://t.me/c/[censored]/[censored]",
    "fi.mau.telegram.source": {
      "chat_id": [censored],
      "id": [censored],
      "peer_type": "channel",
      "space": [censored]
    },
    "fi.mau.telegram.unsupported": true,
    "format": "org.matrix.custom.html",
    "formatted_body": "<mx-reply><blockquote><a href='https://matrix.to/#/![censored]'>In reply to</a> <a href='https://matrix.to/#/@[censored]:[censored]'>@[censored]:[censored]</a><br/><blockquote>\n<p>[censored: This is some text of a quoted message]</p>\n</blockquote></mx-reply>This message is not supported on your version of Mautrix-Telegram. Please check https://github.com/mautrix/telegram or ask your bridge administrator about possible updates.",
    "m.relates_to": {
      "m.in_reply_to": {
        "event_id": "$[censored]"
      }
    },
    "msgtype": "m.notice"
  },
  "origin_server_ts": [censored],
  "sender": "@telegram_[censored]:[censored]",
  "type": "m.room.message",
  "unsigned": {
    "age": 265
  },
  "event_id": "$[censored]",
  "room_id": "![censored]:[censored]"
}
```

People whose messages are like this say that they have quoted a part of a message.

Bridge version according to `!tg version`: [mautrix-telegram](https://github.com/mautrix/telegram) [0.14.2](https://github.com/mautrix/telegram/releases/tag/v0.14.2).

Regards!

**Comment by Xiretza** [comment]
These are partial quotes, implemented in https://github.com/mautrix/telegram/commit/90a8583ad0f2b487269d634e8700175e6c58b6dc.

**Comment by dreirund** [comment]
> These are partial quotes, implemented in [90a8583](https://github.com/mautrix/telegram/commit/90a8583ad0f2b487269d634e8700175e6c58b6dc).

But not yet in release?
